### ISSUE TITLE
Flaky test on TestSimpleExample

### ISSUE THREAD

**Issue by romanini-ciandt** [terraform-google-modules/terraform-google-kms#132]
### TL;DR

In [TestSimpleExample](https://github.com/terraform-google-modules/terraform-google-kms/blob/32d859d35e00d2d3f8eb2a06febe45c7a9fdd43c/test/integration/simple_example/simple_example_test.go#L26) there is an assertion that uses the first element of the gcloud command (`kms keyrings list`) output. This gcloud command, by default, is sorted by keyring's `name` and causes flaky error at the test when running multiple times locally.
Reason: because keyring is not "removable", if you run the same test multiple times, a bunch of keyrings will be generated and the test will start to fail due to a wrong comparison. (it will compare with the first keyring returned, which is not always the correct)

### Expected behavior

The gcloud command should return the expected keyring instead

### Terraform Configuration

```hcl
N/A
```

### Terraform Version

```sh
N/A
```
